src/scripts/index.js

```diff
@@ -0,0 +1,79 @@
+const menu = document.getElementById('menu');
+const mobileNav = document.getElementById('openNav');
+
+menu.addEventListener('click', () => {
+  mobileNav.style.display = 'flex';
+});
+
+const mobileNavClose = document.getElementById('close');
+mobileNavClose.addEventListener('click', () => {
+  mobileNav.style.display = 'none';
+});
+
+const speakers = [
+  {
+    name: 'Barr Moses',
+    profession: 'Co-founder & CEO at Monte Carlo',
+    details: 'Barr is co-founder of data reliability company Monte Carlo. Monte Carlo builds and operates an end-to-end data observability platform, and is backed by Accel, GGV, Redpoint, Iconiq Growth, Salesforce Ventures, IVP, and other top Silicon Valley investors.',
+    img: 'url("https://web-summit-avenger.imgix.net/production/avatars/original/f7c2bb7417bcecc3fa80736d537400710d81f3db.jpeg?ixlib=rb-3.2.1&auto=format&fit=crop&crop=faces&w=300&h=300")',
+  },
```

Comment:
Your Implementation | Original Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/181543491-40628826-4992-40ee-a7aa-77e2c70b3748.png) | ![](https://user-images.githubusercontent.com/63087491/181543693-fbf6365e-6889-435c-a69c-ecaeb103d9a9.png)


- [Optional] Let's take a keen look at the two designs(Yours and Original). Can you see some differences? For example, the transparent background missing from your card, and the `details` font-weight, `name` value not align with the image. Margins are slightly different